src/Apps/Order/redirects.tsx

```diff
@@ -219,7 +219,7 @@ export const newDetailsEnabled = ({
   featureFlags,
 }: Order2RedirectArgs): boolean => {
   return !!(
-    order.mode === "BUY" &&
+    (order.mode === "BUY" || order.mode === "OFFER") &&
```

Comment:
🤯

Reply:
Yea - I left this line here for now even though it's technically all orders so we can tweak what we want to see if we need to. But theoretically this whole function will be removed once we enable feature flag for all!